test/TestAssets/LegacySettingsUnitTestProject/LegacySettingsUnitTestProject.csproj

```diff
@@ -0,0 +1,36 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project InitialTargets="CopyFiles" Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TestPlatformRoot Condition="$(TestPlatformRoot) == ''">..\..\..\</TestPlatformRoot>
+    <TestProject>true</TestProject>
+  </PropertyGroup>
+  <Import Project="..\..\..\scripts\build\TestAssets.props" />
+  <PropertyGroup>
+    <OutputType>library</OutputType>
+    <TargetFramework>net451</TargetFramework>
+    <AssemblyName>LegacySettingsUnitTestProject</AssemblyName>
+  </PropertyGroup>
+  <ItemGroup>
+    <Reference Include="DependencyAssemblyForTest">
+      <HintPath>DependencyAssembly\DependencyAssemblyForTest.dll</HintPath>
+      <Private>false</Private>
+    </Reference>
+    <Reference Include="System" />
+    <Reference Include="Microsoft.CSharp" />
+    <Reference Include="System.Xml" />
+    <Reference Include="Microsoft.VisualStudio.QualityTools.UnitTestFramework" />
+  </ItemGroup>
+  <ItemGroup>
+    <PackageReference Include="Microsoft.VisualStudio.QualityTools">
+      <Version>15.5.0-preview-1046721</Version>
```

Comment:
Move this to latest.

Reply:
Move this to prop file and change in other mstest v1 based test projects?

Reply:
done